docs/rollup-boost-ha.md

```diff
@@ -0,0 +1,328 @@
+
+### Failure Scenarios
+
+<details>
+<summary> Failure Scenarios</summary>
+
+Below is a high level summary of how each failure scenario is handled. All existing failure modes assumed by upstream `op-conductor` are maintained:
+
+| Failure Scenario | Category | Scenario and Solution |
+| --- | --- | --- |
+| Leader `rollup-boost` Fails | Rollup Boost Failure | Leader sequencer `rollup-boost` fails, causing `op-conductor`s sequencer health checks to fail, notifying conductor to elect a new leader. This failure mode is the same as a typical leader sequencer failure. Once the sequencer `rollup-boost` recovers, the sequencer will rejoin the sequencer set as a follower. |
+| Follower `rollup-boost` Fails | Rollup Boost Failure | Follower sequencer `rollup-boost` fails. The leader sequencer is unaffected. Once `rollup-boost` is recovers, the sequencer will rejoin the set as a follower. |
+| `n-1` Builder Failures | Builder Failure | External block production will continue as normal. As long as there is one external block builder available, the healthy instance will continue to produce blocks. Once the builders recover or are manually brought back online, they will join the builder set and continue to operate as normal. |
+| All Builders Fail | Builder Failure | `rollup-boost` will forward FCU with payload attributes to the builder endpoints as normal. Upon a `get_payload` request, if there are no healthy builders available, no builder block will be returned. `rollup-boost`'s block selection logic will time out and fallback to the default execution client's block |
+|Multiple Sequencers Send FCUs w/ Payload Attrs|OP Conductor Failure|In this failure mode, the `stopSequencer` call either fails to send or is delayed causing two sequencers to send FCUs with payload attributes to the builder network. It is unclear what affect this may have depending on the execution client implementation.|
+
+</details>
+
+### Pros and Cons
+
+<details>
+<summary>Pros/Cons</summary>
+
+#### Pros
+
+- Supports builder redundancy and high availability  
+- Compatible with existing `op-conductor`  
+- Resilient to individual builder failures and rolling deployments  
+- Flexible setup allowing chains can choose single, multiple, or third-party builders  
+
+#### Cons
+
+- No strict guarantee builders only receive one FCU with payload attributes.
+- Block selection logic adds complexity to `rollup-boost`  
+
+</details>
+
+<br/>
+
+## Builder Load Balancer
```

Comment:
same issue as the `n` builder solution, we need to choose builder deterministically (at least within a block) so that we know the exact flashblocks to stream

Reply:
Sounds good, as mentioned in a previous reply the design doc has been updated to focus on 1:1 Rollup Boost <-> Builder HA deployments.